Makes ventcrawling time-taken customizable per-mob, increases the delay for morphs.

TsunamiAnt opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

This change introduces a `ventcrawl_delay` variable to control the time it takes for a creature to enter a vent.

Morphs now have a longer delay than other vent crawlers. We may want to adjust the time taken for others as well, but that is not the focus of this PR.

Additionally we lower headspider delay from standard 2.5 to 2, due to bacon's request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Ventcrawling is a profoundly busted ability that, when utilized correctly, is basically impossible to work against. Welding vents is indeed a form of counterplay but requires too much effort and too much coordination to be a valid strategy in most situations currently faced by the average crew.

Morphs specifically are being changed due to their general tankyness and ability for open warfare. Often you don't even have the time to approach a morph before they are in a vent.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.
-->

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

![dreamseeker_Lmw0n9Swpa](https://github.com/user-attachments/assets/b64c1e55-e49a-4f8b-a0b8-5e348080bc65)

![dreamseeker_TMNsN9JeNV](https://github.com/user-attachments/assets/41287d4f-7c81-4e51-b222-36d816338925)

</details>

## Changelog
:cl:
add: Added a way to customize ventcrawling time on mobs.
balance: morphs ventcrawl takes longer now.
balance: headspider ventcrawl is a bit faster.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

TsunamiAnt committed:
Adjusts morph vent crawling delay

Makes morphs take longer to enter vents, balancing their mobility.

This change introduces a `ventcrawl_delay` variable to control the time
it takes for a creature to enter a vent. Morphs now have a longer delay
than other vent crawlers.

TsunamiAnt committed:
Uses SECONDS constant for vent crawl delay.

Uses the SECONDS constant to define the vent crawl delay, improving code readability and consistency.

This change ensures that the vent crawl delay is defined in seconds rather than deciseconds, matching the intended unit of time.

PowerfulBacon commented on code/modules/antagonists/morph/morph.dm:
```diff
@@ -16,6 +14,7 @@
 	status_flags = CANPUSH
 	pass_flags = PASSTABLE
 	ventcrawler = VENTCRAWLER_ALWAYS
+	ventcrawl_delay = 100 // Morphs take longer to enter vents (10 seconds vs 2.5 seconds)
```
```suggestion
	ventcrawl_delay = 5 SECONDS
```
10 seconds feels unreasonably slow and is just always a death sentance for them.

Also use the time macros, and there is no point of having a comment that describes what the line of code is doing.

PowerfulBacon reviewed (commented):
Can we also lower changeling headspiders to 1 second, or maybe 2 seconds?

TsunamiAnt committed:
Update code/modules/antagonists/morph/morph.dm

Co-authored-by: PowerfulBacon <26465327+PowerfulBacon@users.noreply.github.com>

TsunamiAnt committed:
Adds headcrab vent crawl delay.

Adds a delay to the headcrab's vent crawling behavior.

This prevents headcrabs from immediately re-entering vents
after exiting.

TsunamiAnt committed:
Merge remote-tracking branch 'origin/morph' into morph
